recipes/r-gsw/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "1.0-3" %}
+
+package:
+  name: r-gsw
+  version: {{ version.replace("-", "_") }}
+
+source:
+  fn: gsw_{{ version }}.tar.gz
+  url:
+    - http://cran.r-project.org/src/contrib/gsw_{{ version }}.tar.gz
+    - http://cran.r-project.org/src/contrib/Archive/gsw/gsw_{{ version }}.tar.gz
+  sha256: e24c6802467546d616531499c2a98610fdb5c35c8a1e161b6d6b887c4c814331
+
+build:
+  number: 0
+  skip: True  # [win32]
+  script: R CMD INSTALL --build .
```

Comment:
@mingwandroid is it OK to use `script` in `R` recipes? I did notice that you do not use `script` in the recipes in the `r` channel. (I copied this one from there.)